src/app/@ansyn/imagery/public_api.ts

```diff
@@ -1,29 +1,3 @@
-export { IOverlaysCriteriaOptions } from './model/overlay.model';
-
-export { GeoRegisteration } from './model/overlay.model';
-export { ICaseSliderFilterMetadata } from './model/case.model';
-export { ICaseLayersState } from './model/case.model';
-export { OverlayDisplayMode } from './model/case.model';
-export { ICaseFacetsState, ICaseFilter, CaseFilterMetadata } from './model/case.model';
-export { IOverlaysFetchData, IOverlayDrop, IOverlaySpecialObject } from './model/overlay.model';
-export { IPendingOverlay } from './model/overlay.model';
-export { ICase, ICasePreview, IDilutedCase } from './model/case.model';
-export { CaseRegionState, IDataInputFilterValue } from './model/case.model';
-export {
-	CaseGeoFilter,
-	CaseOrientation,
-	CaseTimeFilter, FilterType,
-	ICaseDataInputFiltersState,
-	ICaseTimeState
-} from './model/case.model';
-export { IContextEntity } from './model/case.model';
-export { ICaseBooleanFilterMetadata, ICaseState, IOverlaysManualProcessArgs } from './model/case.model';
-export { ICaseMapsState, IDilutedCaseState, ImageManualProcessArgs } from './model/case.model';
-export { IWorldViewMapState } from './model/case.model';
-export { CaseEnumFilterMetadata } from './model/case.model';
-export { ICaseMapState } from './model/case.model';
-export { IOverlay, IDilutedOverlay, Overlay, IOverlaysCriteria } from './model/overlay.model';
-
 export { MAP_SOURCE_PROVIDERS_CONFIG, IMapSourceProvidersConfig } from './model/base-map-source-provider';
 export { ICaseMapPosition, CaseMapExtent, CaseMapExtentPolygon, ICaseMapProjectedState } from './model/case-map-position.model';
```

Comment:
rename ICaseMapPosition, CaseMapExtent, CaseMapExtentPolygon, ICaseMapProjectedState
remove the "Case" word